bidwire/notifier.py

```diff
@@ -28,6 +28,7 @@ def send_new_bids_notifications(recipient_emails):
     for notifier in notifiers:
         site = notifier.get_site()
         new_bids = bid.get_bids_from_last_n_hours(23, notifier.get_site())
-        notifier.send_new_bids_notification(new_bids)
+        if len(new_bids) > 0:
```

Comment:
FYI, in Python, an empty list will return False in a boolean check, so you can write this as:
```python
if new_bids:
  send_notification(new_bids)
```